packages/istanbul-lib-source-maps/lib/transformer.js

```diff
@@ -82,6 +82,18 @@ class SourceMapTransformer {
                     locs.push(mapping.loc);
                     mappedHits.push(hits[i]);
                 }
+                // Check if this is an implicit else
+                else if (
+                    branchMeta.type === 'if' &&
+                    i > 0 &&
+                    loc.start.line === undefined &&
+                    loc.start.end === undefined &&
+                    loc.end.line === undefined &&
+                    loc.end.end === undefined
+                ) {
+                    locs.push(loc);
+                    mappedHits.push(hits[i]);
+                }
```

Comment:
For reference:

https://github.com/istanbuljs/istanbuljs/blob/fb8cb4bb99c9e5d36c8b011338d2ab5d3c72ff9c/docs/raw-output.md?plain=1#L49

https://github.com/istanbuljs/istanbuljs/blob/fb8cb4bb99c9e5d36c8b011338d2ab5d3c72ff9c/docs/raw-output.md?plain=1#L52

https://github.com/istanbuljs/istanbuljs/blob/fb8cb4bb99c9e5d36c8b011338d2ab5d3c72ff9c/packages/istanbul-reports/lib/html/annotator.js#L147-L154